tests/utils/test_ellipsoid_utils.py

```diff
@@ -0,0 +1,92 @@
+"""Unit tests for functions in ellipsoid utils file.
+
+Authors: Adi Singh
+"""
+import gtsfm.utils.ellipsoid as ellipsoid_utils
```

Comment:
let's group these per the [Google style guide](https://google.github.io/styleguide/pyguide.html#313-imports-formatting):

```python
import unittest

import numpy as np
import numpy.testing as npt

import gtsfm.utils.ellipsoid as ellipsoid_utils
```